omise/payment_methods/_payment_method.php

```diff
@@ -72,7 +72,7 @@ public static function availableForCurrency($code)
         return count(static::$restrictedToCurrencies) ? in_array($curr, static::$restrictedToCurrencies) : true;
     }
 
-    public static function getLink($method, $params = [], $controller = 'paymentmethod', $ssl = false) 
+    public static function getLink($method, $params = [], $controller = 'paymentmethod', $ssl = true) 
```

Comment:
Should the default value be `null` instead? Or is there any particular reason why do we want the default to be `true`? (just checked the `getModuleLink` method from PrestaShop's `LinkCore` class, it leaves its default value as null)

Reply:
I did think that so it would presumably follow the prestashop settings, but I deferred to what the plugin was doing before the refactor, which was making all the payment method links HTTPS

Reply:
okay